src/Tests/Microsoft.DotNet.TemplateLocator.Tests/GivenAnTemplateLocator.cs

```diff
@@ -47,7 +47,7 @@ public void ItShouldReturnListOfTemplates()
             string templateNupkgPath = Path.Combine(templatePacksDirectory, "xamarin.android.templates.1.0.3.nupkg");
             File.WriteAllText(templateNupkgPath, "");
 
-            var result = _resolver.GetDotnetSdkTemplatePackages("5.0.102", _fakeDotnetRootDirectory);
+            var result = _resolver.GetDotnetSdkTemplatePackages("5.0.102",_fakeDotnetRootDirectory, userHome: Path.GetTempPath());
```

Comment:
I think if we move the logic to get the user home into the workload manifest reader and resolver, then we won't have to pass a fake user home path in all of these tests.